src/app/footer-navbar/footer-navbar.component.css

```diff
@@ -8,8 +8,6 @@
   justify-content: space-between;
   align-items: center;
   margin-bottom: 0px;
-  position: fixed;
-  bottom: 0px;

```

Comment:
The position attributes should be independent of the child component I inherit, they should depend only on parent component. when I use the same component in other parent components, I should not have any internal css factors of the child component affecting the position in the parent component, there should be independency to apply my own css  in each parent component I inherit this. so where ever you want to fix the component as fixed , write this css only in the parent component but not in the child component, such that we could have other positions defined in other parent components.